articles/020_ros_with_dds.md

```diff
@@ -151,6 +151,8 @@ It is available on [Github](https://github.com):
 ADLINK's implementation comes with a basic, functioning build system and was fairly easy to package.
 OpenSplice appears to be the number two DDS implementation in use, but that is hard to tell for sure.
 
+ADLINK also contributes to Eclipse Cyclone DDS [https://github.com/eclipse-cyclonedds/cyclonedds] (https://www.adlinktech.com/en/CycloneDDS).  Cyclone DDS is licensed under the Eclipse Public License - v 2.0 ( https://www.eclipse.org/legal/epl-2.0/).
```

Comment:
Nitpick: please start each sentence on a new line (following the style guide). Also remove the extra space after the open parenthesis.